src/lib/code-sample/javascript.ts

```diff
@@ -8,7 +8,7 @@ export const createJavascriptRequest = (
   _context: Context,
 ): string => {
   const parts = request.path.split('/')
-  const isWithoutParams = Object.keys(request.parameters).length === 0
+  const isWithoutParams = Object.keys(request.parameters ?? {}).length === 0
```

Comment:
Instead of doing this in every implementation we only need to do this in one spot in the zod schema with `.default({})`

Reply:
https://github.com/seamapi/blueprint/blob/main/src%2Flib%2Fcode-sample%2Fschema.ts#L30

Reply:
Hmm, makes sense. The schmea should throw if null is passed though, can we try this and just not support passing `null`?

```
parameters: z.record(z.string().min(1), JsonSchema).optional().default({}),
```

Reply:
888fc330d8154e10cc9a5c051765acb5bd91472b